src/briefcase/platforms/web/static.py

```diff
@@ -453,17 +453,25 @@ class StaticWebPublishCommand(StaticWebMixin, PublishCommand):
 
 
 class StaticWebDevCommand(StaticWebMixin, DevCommand):
-    description = "Run a static web project in development mode. (Work in progress)"
+    description = "Run a static web project in development mode.(Work in progress...)"
```

Comment:
The old text was fine.
```suggestion
    description = "Run a static web project in development mode. (Work in progress)"
```